tests/testKron.py

```diff
@@ -102,17 +94,19 @@ def makeGalaxy(width, height, flux, a, b, theta, dx=0.0, dy=0.0, xy0=None, xcen=
             Iuu += val*u**2
             Ivv += val*v**2
 
-    Iuu /= I; Ivv /= I
+    Iuu /= I
+    Ivv /= I
 
     exp = afwImage.makeExposure(afwImage.makeMaskedImage(gal))
-    exp.getMaskedImage().getVariance().setXY0(exp.getXY0()) # workaround #2577
+    exp.getMaskedImage().getVariance().setXY0(exp.getXY0())  # workaround #2577
```

Comment:
Looks like the relevant ticket is https://dev.lsstcorp.org/trac/ticket/2577 and it seems that was fixed.